### ISSUE TITLE
Sneak and LeftClick shows normal inventory (purpur)

### ISSUE THREAD

**Issue by sdosker** [DidlDimpfe/Automatic-Crafting-Table-Plugin#7]
Recently i had to remove paper and i'm using purpur from now, the plugin was working fine on paper.

On purpur, i'm not sure if there's an configuration that i could change, but when i try to open the automatic inventory of the crafting table, it simply opens the normal one.

Purpur server version tested: (build_2023) git-Purpur-2023 (MC: 1.20.1) (Implementing API version 1.20.1-R0.1-SNAPSHOT) (Git: 41b121b on HEAD)

**Comment by DidlDimpfe** [comment]
When you shift and left click a crafting table it cannot open the normal one. Did you right click?

**Comment by sdosker** [comment]
Yeah, it was opening the normal crafting table with recipe book and everything...
Im trying to get more info to try to reproduce easier, because after restarting the server, nothing happen with sneak+left click

**Comment by sdosker** [comment]
I've tried to get more info but there was no way...
My hypothesis is that the configuration corrupted somehow, because after simply deleting the configuration file (from the plugin folder) and restarting the server, it went back to working as if nothing had happened... 
I tried to put back the paper, restore backups, install purpur again without its settings, but the bug did not happen anymore ...
In summary, apparently deleting the plugin settings was the solution, but why it was necessary I couldn't figure out because there was no trace of errors on the console
